### DISCUSSION TITLE
Calculate Perceptual Index (PI) of BasicVSR and BasicVSR++ results

### DISCUSSION THREAD

**Discussion by JayChen9909** [open-mmlab/mmediting#1305] [How-to] [answered]
Since I recognized that both of the super resolution models BasicVSR and BasicVSR++ are using PSNR and SSIM to evaluate the results, did anybody try to use PI to evaluate the results based on these models? If so, which file of datasets have you guys used as the output results and which part have you used as the validation/test dataset to go through the PI calculation from PIRM2018 method?

**Comment by zengyh1900** [comment]
@LeoXing1996 
Please check this issue.

**Comment by LeoXing1996** [comment, marked as answer]
Hey @JayChen9909 , in MMEditing has not supported PI evaluation for BasicVSR and BasicVSR++. 
If you have any implementation or idea for evaluating PI, NIQE, and NRQM metrics, we can have a discussion. 
You can also directly create a PR to share your implementation.

BTW, MMEditing has pre-released [1.x version](https://github.com/open-mmlab/mmediting/tree/1.x) in recent days, which is based on [MMEngine](https://github.com/open-mmlab/mmengine) and [MMCV 2.x](https://github.com/open-mmlab/mmcv/tree/2.x). We encourage the community to develop based on the 1.x version.
